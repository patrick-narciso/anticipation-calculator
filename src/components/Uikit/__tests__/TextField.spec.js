// eslint-disable-next-line no-unused-vars
import { dom } from 'isomorphic-jsx';
import { TextField } from '@Uikit';

test('Should be render TextField Component without extraMessage correctly', () => {
  expect(<TextField id="amount" type="text" required="required" label="Informe o valor da venda*" />).toMatchSnapshot();
});

test('Should be render TextField Component with extraMessage correctly', () => {
  expect(
    <TextField
      id="installments"
      extraMessage="MÃ¡ximo de 12 parcelas"
      type="number"
      required="required"
      label="Em quantas parcelas*"
    />
  ).toMatchSnapshot();
});
