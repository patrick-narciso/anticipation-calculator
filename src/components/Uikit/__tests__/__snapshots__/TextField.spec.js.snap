// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be render TextField Component with extraMessage correctly 1`] = `"<div class=\\"css-11k5a09-wrapper\\"><label class=\\"css-18tgiz1-labelInput\\" for=\\"installments\\">Em quantas parcelas*</label><div class=\\"css-1rdjunr-inputWrapper\\"><div class=\\"css-memc06-wrapper\\"><input id=\\"installments\\" class=\\"css-1u7nkg2-fieldInput\\" name=\\"installments\\" type=\\"number\\" onchange=\\"undefined\\" value=\\"\\"></input></div><span class=\\"css-1vd4a9j-extraInfo\\">MÃ¡ximo de 12 parcelas</span></div></div>"`;

exports[`Should be render TextField Component without extraMessage correctly 1`] = `"<div class=\\"css-11k5a09-wrapper\\"><label class=\\"css-18tgiz1-labelInput\\" for=\\"amount\\">Informe o valor da venda*</label><div class=\\"css-1rdjunr-inputWrapper\\"><div class=\\"css-memc06-wrapper\\"><input id=\\"amount\\" class=\\"css-1u7nkg2-fieldInput\\" name=\\"amount\\" type=\\"text\\" onchange=\\"undefined\\" value=\\"\\"></input></div></div></div>"`;
